src/components/ArticleCard.jsx

```diff
@@ -29,6 +56,16 @@ const ArticleCard = () => {
           <div className="author-parent">
             <h3 className="card-author"> written by {article.author}</h3>
             <h4 className="card-topic"> category {article.topic}</h4>
+            <h4 className="card-votes"> likes {votes}</h4>
+            <button
+              type="button"
+              className="like-button"
+              onClick={() => {
+                handleVote();
+              }}
+            >
+              +
```

Comment:
This button works well, it would be good though if the button changed to reflect what it was doing, or that it is being toggled. For example it could say '-' instead of '+' when pressing it decreases the votes to its original value, or it could turn a different colour when the vote has been added 
This would make more obvious for the user to understand its function ✅